code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -55,6 +57,41 @@
 /mob/living/simple_animal/hostile/Life()
 	if(timestopped)
 		return 0 //under effects of time magick
+	if(!client || deny_client_move) //Ventcrawling stuff
+		if(entry_vent)
```

Comment:
nothing even sets this because nothing is calling get_vent() to begin with...

need to add some AI logic to decide to ventcrawl sometimes

Reply:
It's called in spiderling and gremlin types